src/platforms/apple/common/usage.mdx

```diff
@@ -50,6 +50,55 @@ This integration is the core part of the SDK. It hooks into all signal and excep
 
 This integration will swizzle some methods to create breadcrumbs e.g.: for `UIApplicationDidReceiveMemoryWarningNotification`, `sendAction:to:from:forEvent:` (UI interactions) or `viewDidAppear:` those breadcrumbs will be attached to your events.
 
+## In App Stack Frames
+
+To highlight essential frames of your stacktrace, the Sentry UI only displays frames related to your application by default. Sentry marks such frames as inApp.
```

Comment:
```suggestion
To highlight essential frames of your stacktrace, sentry.io displays frames related to only your application by default. Sentry marks such frames as inApp.
```

Reply:
Do we have a special section about InApp frames in the docs shared across all SDKS?

Reply:
We have this one: https://docs.sentry.io/platforms/apple/data-management/event-grouping/stack-trace-rules/#mark-in-app-frames

I suggest we move this doc elsewhere other than `usage`.

It's a very generic kind of page IMHO.

Reply:
The information on this page is not generic, though. It's very specific to Apple. We could make it top level or put it below Advanced Usage, for example.

Reply:
before merging this, what's about to the suggested page https://docs.sentry.io/platforms/apple/data-management/event-grouping/stack-trace-rules/#mark-in-app-frames but adding the `supported` flag which is gonna make it visible only for `apple` anyway? because this page looks like the best place to be anyway.